tests/acceptance/features/webUIFiles/batchAction.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Visibility of the batch actions menu
+  As a user
+  I want to see the batch action menu options
+  So that the batch action menu options of the selected file or folder are visible to me
+
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+    And user "Alice" has created folder "simple-folder"
+    And user "Brian" has created folder "brain-folder"
```

Comment:
I think it's better to place these steps in the scenarios that are specifically using them